python-package/lightgbm/basic.py

```diff
@@ -409,7 +418,7 @@ def predict(self, data, num_iteration=-1,
                                                predict_type)
         elif isinstance(data, DataFrame):
             preds, nrow = self.__pred_for_np2d(data.values, num_iteration,
-                                               predict_type)
+                                               predict_type, early_stop_instance_handle)
```

Comment:
this was fixed in the new version, the python code was not ready yet at the time you did the review